src/NBitcoin/ChainBase.cs

```diff
@@ -161,6 +189,11 @@ public ChainedBlock FindFork(BlockLocator locator)
             return this.FindFork(locator.Blocks);
         }
 
+        /// <summary>
+        /// Enumerate chain after given block hash to genesis block.
+        /// </summary>
+        /// <param name="blockHash">Block hash to enumerate after.</param>
+        /// <returns>Enumeration of chained block headers after given block hash.</returns>
```

Comment:
This feels to me like it could be an extension method, perhaps it might be a good idea to see what other methods can be pulled as extensions (not this PR)